docroot/themes/custom/uids_base/scss/components/search-results.scss

```diff
@@ -79,7 +82,7 @@
   padding-right: 0;
 }
 #search-results .gsc-url-top .gs-visibleUrl, #search-results .gsc-url-bottom .gs-visibleUrl {
-  color: $link-color;
+  color: $blk;
```

Comment:
@ethanstjohn Can you change this to `$brand-cool-gray`?  I think having this as gray will help further distinguish that this url is not active.